misk/src/main/kotlin/misk/digester/Histogram.kt

```diff
@@ -0,0 +1,8 @@
+package misk.digester
+
+interface Histogram {
+
+    fun record(vararg labelValues: String): Any
```

Comment:
what does this method do if you don't provide a duration to record?

Reply:
This was to support the tests (like the ones in MetricsInterceptorTest). I'll refactor the tests to send a duration as well.

Reply:
Actually I think the method existed to support a builder pattern with the prometheus client